### ISSUE TITLE
Cannot read data from ADIOS1 BP file which has no subfiles

### ISSUE THREAD

**Issue by pnorbert** [ornladios/ADIOS2#568]
ADIOS 1.x MPI transport produces a BP file with data in it. No subfiles. The subfile index for variables is -1. 

ADIOS 2 misinterprets that -1 and still tries to read from a subfile:

ERROR: couldn't open file heat.bp.dir/heat.bp.4294967295, check permissions or path existence, in call to POSIX open
: iostream error
adios@adiosVM:~/work/adiosvm/Tutorial/heat_transfer$ ls -l heat.bp 
-rw-rw-r-- 1 adios adios 2409312 Apr 23 09:43 heat.bp

**Comment by pnorbert** [comment]
To reproduce, look at adiosvm Tutorial, old heat_transfer/ example
```
$ make adios1
edit heat_transfer.xml and make sure transport method for group heat is set to MPI
  <method group="heat" method="MPI"/>
$  mpirun -n 1 ./heat_transfer_adios1   heat   1 1   4 5   1  1
 Process number        : 1 x 1
 Array size per process at first step: 4 x 5
Step    1:
Total runtime =  0.001s
$ ls -l  heat.bp*
-rw-rw-r-- 1 adios adios 3481 Apr 23 10:15 heat.bp
$ adios2_reorganize heat.bp r.bp BPFile "" BPFile "" 
...
ERROR: couldn't open file heat.bp.dir/heat.bp.4294967295, check permissions or path existence, in call to POSIX open
: iostream error
```

**Comment by chuckatkins** [comment]
Fixed by #1056
